### PR TITLE
Project setting data retention

### PR DESCRIPTION
**Pull request by pivnicek**
ref: https://github.com/keboola/connection/issues/1040

Set data retention period attribute for project.

### CODE REVIEW

File: tests/ProjectsTest.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ private function initTestProject($organizationId)
         $this->assertArrayHasKey('limits', $foundProject);
         $this->assertTrue(count($foundProject['limits']) > 1);
         $this->assertArrayHasKey('metrics', $foundProject);
-        $this->assertEquals('mysql', $foundProject['defaultBackend']);
+        $this->assertEquals('snowflake', $foundProject['defaultBackend']);
```
Comment (Halama) [inline tests/ProjectsTest.php]: @pivnicek how did you make this working? It'll required also changes in default templates. But we are also removing mysql backend we'll have to do this changes so I'll prepare it

Comment (pivnicek) [reply tests/ProjectsTest.php]: ah, yes, I changed the default backend locally to snowflake in my template.

**Comment by pivnicek** [comment]
one TODO here will be to update the project templates with `dataRetentionTimeInDays`

**Comment by Halama** [comment]
yes, but these will be inialized with 1 which is ok, we can change it later.

**Comment by Halama** [comment]
I am not sure how to change templates for dev and tests. Because templates are not version, there were some init data in migrations but now it is set up manually in each stack.

**Comment by pivnicek** [comment]
A fresh migration may be the best way

**Comment by pivnicek** [comment]
something like `truncate` and then `insert`.  

or we could add project template management to this api :-)

**Comment by Halama** [comment]
yes, return it back to SCM by similar migration https://github.com/keboola/connection/blob/master/sql/migrations/accounts/Version20160224150058.php
All stack will have the same templates which is imho ok. So I'l grab templates and limits from current us-east stack. And I'll do it as separate PR preceding this one.

**Pull request merged**